internal/provider/resource_gitlab_group.go

```diff
@@ -148,11 +148,33 @@ var _ = registerResource("gitlab_group", func() *schema.Resource {
 				Computed:    true,
 				Sensitive:   true,
 			},
-			"prevent_forking_outside_group": {
-				Description: "When enabled, users can not fork projects from this group to external namespaces.",
+			"membership_lock": {
+				Description: "Prevent adding new members to projects within this group. Requires GitLab Premium.",
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Default:     false,
+			},
+			"file_template_project_id": {
+				Description: "The ID of a project to load custom file templates from. Requires GitLab Premium.",
+				Type:        schema.TypeInt,
+				Optional:    true,
+			},
+			"shared_runners_minutes_limit": {
+				Description: "Maximum number of monthly CI/CD minutes for this group. Can be nil " +
```

Comment:
I think it makes sense to use back ticks here :)

Reply:
fixed, thanks for the reminder 😃 , my Java background leaking out here!